### PR TITLE
add splitting to doc

### PR DESCRIPTION
**Pull request by LaraAS**
Add splitting of a neuron to docs

### CODE REVIEW

File: docs/developer-docs/daos/sns/managing/making-proposals.mdx [hunk 1]
```diff
@@ -601,6 +601,62 @@ quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
...
+```candid
+  type ManageLedgerParameters = record {
+    transfer_fee : opt nat64
+  };
+```
+
+Example in bash:
+
+```bash
+TODO
```
Comment (LaraAS) [inline docs/developer-docs/daos/sns/managing/making-proposals.mdx:621]: @daniel-wong-dfinity-org could you please fill this in?

File: docs/developer-docs/daos/sns/managing/making-proposals.mdx [hunk 2]
```diff
@@ -601,6 +601,62 @@ quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
...
+  LOG_VISIBILITY_UNSPECIFIED = 0;
+  LOG_VISIBILITY_CONTROLLERS = 1;
+  LOG_VISIBILITY_PUBLIC = 2;
+}
+```
+
+#### Putting it together
+
+```
+TODO
```
Comment (LaraAS) [inline docs/developer-docs/daos/sns/managing/making-proposals.mdx:651]: @daniel-wong-dfinity-org could you please fill this in?

File: docs/developer-docs/daos/sns/managing/making-proposals.mdx [hunk 3]
```diff
@@ -601,6 +601,62 @@ quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
...
+```
+
+Example in bash:
+
+```bash
+TODO
```
Comment (LaraAS) [inline docs/developer-docs/daos/sns/managing/making-proposals.mdx:657]: @daniel-wong-dfinity-org could you please fill this in?

**Comment by daniel-wong-dfinity-org** [comment]
👀

File: docs/developer-docs/daos/sns/managing/making-proposals.mdx (hunk 1 above)
Comment (daniel-wong-dfinity-org) [reply docs/developer-docs/daos/sns/managing/making-proposals.mdx:621]: quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(                                  
    record {                                             
        title = "Change Our Ledger Fee to 314_159 e8s";
        url = "https://sns-examples.com/proposal/42";
        summary = "Change Our Ledger Fee to 314_159 e8s.";
        action = opt variant {
            ManageLedgerParameters = record {
                transfer_fee = opt 314_159;
            }                                    
        }
    }
)' --canister-ids-file openchat_canister_ids.json > message.json

quill send message.json

File: docs/developer-docs/daos/sns/managing/making-proposals.mdx (hunk 3 above)
Comment (daniel-wong-dfinity-org) [reply docs/developer-docs/daos/sns/managing/making-proposals.mdx:657]:
```bash
quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
    record {
        title = "Set the Memory Allocation of the Widget Canister to 314_159 Bytes";
        url = "https://sns-examples.com/proposal/42";
        summary = "Set the memory allocation of the Widget Canister to 314_159 bytes.";
        action = opt variant {
            ManageDappCanisterSettings = record {
                canister_ids = vec { principal "WIDGET-CANISTER-ID" };
                memory_allocation = opt 314_159;
            }
        }
    }
)' --canister-ids-file <PATH_TO_CANISTER_IDS_JSON_FILE> > message.json

quill send message.json
```

File: docs/developer-docs/daos/sns/managing/making-proposals.mdx (hunk 2 above)
Comment (daniel-wong-dfinity-org) [reply docs/developer-docs/daos/sns/managing/making-proposals.mdx:651]: Not sure why you have two code blocks in this section. Usually, there is just one. Is this a mistake?

Comment (LaraAS) [reply docs/developer-docs/daos/sns/managing/making-proposals.mdx:651]: I think if there is a composed field there was usually an intermediate state just to have them together.. but I think this is not top prio

**Review by jessiemongeon1 — APPROVED** [review]

**Pull request merged**